src/sentry/static/sentry/app/components/modals/inviteMembersModal/renderEmailValue.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled, {css} from 'react-emotion';
+import {Value} from 'react-select';
+
+import InlineSvg from 'app/components/inlineSvg';
+import space from 'app/styles/space';
+import Tooltip from 'app/components/tooltip';
+import LoadingIndicator from 'app/components/loadingIndicator';
+
+import {InviteStatus} from '.';
+
+function renderEmailValue(status: InviteStatus[string], valueProps) {
```

Comment:
Should `valueProps` be typed?

Reply:
We can type this once we have types from react-select.